code/style.css

```diff
@@ -1,5 +1,84 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
 body {
-  background-color: lightblue;
+  font-family: 'Quicksand', sans-serif;
+  font-style: normal;
 }
```

Comment:
To make your content go all the way to the edges (to remove the surrounding white space), you can add `margin: 0;` to the body!

<img width="327" alt="Skärmavbild 2020-10-15 kl  11 01 29" src="https://user-images.githubusercontent.com/51529201/96101522-d6d46a00-0ed5-11eb-82c1-9e2e6362d8ee.png">